doc/api/fs.md

```diff
@@ -357,16 +429,165 @@ Stats {
   birthtime: Mon, 10 Oct 2011 23:24:11 GMT }
 ```
 
+### stats.isBlockDevice()
+
+* Returns: {boolean}
+
+Returns `true` if the `fs.Stats` object describes a block device.
+
+### stats.isCharacterDevice()
+
+* Returns: {boolean}
+
+Returns `true` if the `fs.Stats` object describes a character device.
+### stats.isDirectory()
+
+* Returns: {boolean}
+
+Returns `true` if the `fs.Stats` object describes a file system directory.
+### stats.isFIFO()
+
+* Returns: {boolean}
+
+Returns `true` if the `fs.Stats` object describes a first-in-first-out (FIFO)
+pipe.
+
+### stats.isFile()
+
+* Returns: {boolean}
+
+Returns `true` if the `fs.Stats` object describes a regular file.
+
+### stats.isSocket()
+
+* Returns: {boolean}
+
+Returns `true` if the `fs.Stats` object describes a socket.
+
+### stats.isSymbolicLink()
+
+* Returns: {boolean}
+
+Returns `true` if the `fs.Stats` object describes a symbolic link.
+
+*Note*: This method is only valid when using [`fs.lstat()`][]
+
+
+### stats.dev
+
+* Value: {number}
+
+The numeric identifier of the device containing the file.
+
+### stats.ino
+
+* Value: {number}
+
+The file system specific "Inode" number for the file.
+
+### stats.mode
+
+* Value: {number}
+
+A bit-field describing the file type and mode.
+
+### stats.nlink
+
+* Value: {number}
+
+The number of hard-links that exist for the file.
+
+### stats.uid
+
+* Value: {number}
+
+The numeric user identifier of the user that owns the file (POSIX).
+
+### stats.gid
+
+* Value: {number}
+
+The numeric group identifier of the group that owns the file (POSIX).
+
+### stats.rdev
+
+* Value: {number}
+
+A numeric device identifier if the file is considered "special".
+
+### stats.blksize
+
+* Value: {number}
+
+The file system block size for i/o operations.
+
+### stats.blocks
+
+* Value: {number}
+
+The number of blocks allocated for this file.
+
+### stats.atimeMs
+
+* Value: {number}
+
+The timestamp indicating the last time this file was accessed expressed in
+milliseconds since the POSIX Epoch.
+
+### stats.mtimeMs
+
+* Value: {number}
+
+The timestamp indicating the last time this file was modified expressed in
+milliseconds since the POSIX Epoch.
+
+### stats.ctimeMs
+
+* Value: {number}
+
+The timestamp indicating the last time this files status was changed expressed
+in milliseconds since the POSIX Epoch.
+
+### stats.birthtimeMs
+
+* Value: {number}
+
+The timestamp indicating the creation time of this file expressed in
+milliseconds since the POSIX Epoch.
+
+### stats.atime
+
+* Value: {Date}
+
+The timestamp indicating the last time this file was accessed.
+
+### stats.mtime
+
+* Value: {Date}
+
+The timestamp indicating the last time this file was modified.
+
+### stats.ctime
+
+* Value: {Date}
+
+The timestamp indicating the last time this files status was changed.
```

Comment:
:%s/files/file

Reply:
`this files` -> `the file` or `this file's`